app1.js

```diff
@@ -13,7 +13,7 @@ function getUserName() {
     document.write(text);
 }
 
-function dealinAbsolutes() {
+function dealInAbsolutes() {
```

Comment:
Should be 'dealInAbsolutes`.  Note the capital 'I'.  That being said, could be more descriptive. And think about the call out above about functions that do one and one thing only.